src/lazy/encoder/text/v1_1/value_writer.rs

```diff
@@ -319,6 +322,68 @@ impl<'value, W: Write + 'value> MakeValueWriter for TextEExpWriter_1_1<'value, W
     }
 }
 
-impl<'value, W: Write + 'value> EExpWriter for TextEExpWriter_1_1<'value, W> {
+impl<'eexp, W: Write + 'eexp> EExpWriter for TextEExpWriter_1_1<'eexp, W> {
+    type ExprGroupWriter<'group>
+        = TextExprGroupWriter<'group, W>
+    where
+        Self: 'group;
+
+    fn expr_group_writer(&mut self) -> IonResult<Self::ExprGroupWriter<'_>> {
+        TextExprGroupWriter::new(
+            self.container_writer.writer,
+            self.container_writer.depth,
+            self.container_writer.container_type.into(),
+            " ",
+        )
+    }
     // Default SequenceWriter methods
 }
+
+pub struct TextExprGroupWriter<'group, W: Write> {
+    // There is no expr group writer in 1.0 to which we can delegate,
+    // but we can re-use the TextContainerWriter_1_0 for a lot of the formatting.
+    container_writer: TextContainerWriter_1_0<'group, W>,
```

Comment:
🪧 The text expression group writer is (for the moment) basically just a `SExpWriter` with different opening text. Later, we'll validate that method calls on the expr group writer align with the corresponding `Parameter`'s encoding.